odoo/addons/base/models/res_lang.py

```diff
@@ -334,6 +334,22 @@ def format(self, percent, value, grouping=False, monetary=False):
 
         return formatted
 
+    def action_activate_langs(self):
+        """ Activate the selected languages """
+        for lang in self.filtered(lambda l: not l.active):
+            lang.toggle_active()
+        message = _("The languages that you selected have been successfully installed. Users can choose their favorite language in their preferences.")
```

Comment:
How about a "click here to switch to this language" instead (when there is only one lang)?